tests/negative/Internal/LoopingCoercion.juvix

```diff
@@ -0,0 +1,21 @@
+module LoopingCoercion;
+
+type Unit := unit;
+
+trait
+type T1 A := mkT1 {pp : A -> A};
+
+trait
+type T2 A := mkT2 {pp : A -> A};
+
+coercion
+fromT1toT2 {A} {{T1 A}} : T2 A := mkT2@{
+  pp := T1.pp
+};
+
+coercion
```

Comment:
I think the negative tests need to. be updated to use the new keyword and then we can merge this.

Reply:
Yes, thanks, I forgot about them